### PR TITLE
Add app config tab

### LINKED ISSUE #1159: Proposal: Add Application Configuration Information
First up, thanks for the great tool. It's saved me so much time over the years!

**Problem**
I need to see how my application is configured in production or other environments. This configuration information is stored in an URL encoded format in one or more `<meta>` tags in the `<header>` of my app. This config is not easily readable by humans in it's stored format.

**Solution**
Add an application configuration section. This could be added to the info section.

**Details**
The config for the current (3.17.0-beta.1) default output of ember-cli looks like this:
```html
<meta name="my-app/config/environment" content="%7B%22modulePrefix%22%3A%22my-app%22%2C%22environment%22%3A%22development%22%2C%22rootURL%22%3A%22%2F%22%2C%22locationType%22%3A%22auto%22%2C%22EmberENV%22%3A%7B%22FEATURES%22%3A%7B%7D%2C%22EXTEND_PROTOTYPES%22%3A%7B%22Date%22%3Afalse%7D%2C%22_APPLICATION_TEMPLATE_WRAPPER%22%3Afalse%2C%22_DEFAULT_ASYNC_OBSERVERS%22%3Atrue%2C%22_JQUERY_INTEGRATION%22%3Afalse%2C%22_TEMPLATE_ONLY_GLIMMER_COMPONENTS%22%3Atrue%7D%2C%22APP%22%3A%7B%22name%22%3A%22my-app%22%2C%22version%22%3A%22v3.17.0-beta.1%2B7cbe9b8f%22%7D%2C%22exportApplicationGlobal%22%3Atrue%7D">
```
Not fun to read. It can be easily parsed by copying the contents and pasting them into `JSON.parse(decodeURIComponent('encoded content here'))`, but this requires knowledge of the special command and is a bit cumbersome.

### PR DESCRIPTION
**Pull request by hannakim91**
## Description
Added app config tab to info tab. Currently longer objects get cut off. Will look at changing that in the future. 
Closes #1159 
## Screenshots

Screen Shot 2021-05-12 at 9 30 02 AM

### CODE REVIEW

File: app/routes/app-config.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import Route from '@ember/routing/route';
+import TabRoute from 'ember-inspector/routes/tab';
+import { readOnly } from '@ember/object/computed';
+
+export default TabRoute.extend({
+  version: readOnly('config.VERSION'),
+
+  model() {
+    const port = this.port;
+    console.log('port', port);
```
Comment (rwwagner90) [inline app/routes/app-config.js:10]:
```suggestion
```

File: app/routes/app-config.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    return new Promise((resolve) => {
+      port.one('general:emberCliConfig', (message) => {
+        console.log('message', message.emberCliConfig);
```
Comment (rwwagner90) [inline app/routes/app-config.js:13]:
```suggestion
```

File: app/routes/libraries.js [hunk 3]
```diff
@@ -14,6 +14,7 @@ export default TabRoute.extend({
           name: 'Ember Inspector',
           version,
         });
+        console.log('libraries', message.libraries)
```
Comment (rwwagner90) [inline app/routes/libraries.js:17]:
```suggestion
```

File: ember_debug/general-debug.js [hunk 4]
```diff
@@ -27,6 +27,7 @@ export default EmberObject.extend(PortMixin, {
       try {
         let config = JSON.parse(unescape(found.getAttribute('content')));
         this.set('emberCliConfig', config);
+        console.log(config);
```
Comment (rwwagner90) [inline ember_debug/general-debug.js:30]:
```suggestion
```

File: tests/unit/controllers/app-config-test.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Controller | app-config', function(hooks) {
+  setupTest(hooks);
+
+  // TODO: Replace this with your real tests.
+  test('it exists', function(assert) {
```
Comment (rwwagner90) [inline tests/unit/controllers/app-config-test.js:8]: We should probably delete this empty test file.

File: tests/unit/routes/app-config-test.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Route | app-config', function(hooks) {
+  setupTest(hooks);
```
Comment (rwwagner90) [inline tests/unit/routes/app-config-test.js:5]: We should probably delete this empty test file.

**Review by rwwagner90 — CHANGES_REQUESTED** [review]

**Comment by nummi** [comment]
Looks great to me! :+1:

File: app/routes/app-config.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
+import TabRoute from 'ember-inspector/routes/tab';
+import { readOnly } from '@ember/object/computed';
+
+export default TabRoute.extend({
+  version: readOnly('config.VERSION'),
```
Comment (rwwagner90) [inline app/routes/app-config.js:5]:
```suggestion
```

**Review by rwwagner90 — APPROVED** [review]

File: app/routes/app-config.js [hunk 8]
```diff
@@ -0,0 +1,15 @@
+import TabRoute from 'ember-inspector/routes/tab';
+import { readOnly } from '@ember/object/computed';
```
Comment (rwwagner90) [inline app/routes/app-config.js:2]:
```suggestion
```

File: app/routes/app-config.js [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+
+export default TabRoute.extend({
+
```
Comment (rwwagner90) [inline app/routes/app-config.js:5]:
```suggestion
```

**Pull request merged**